examples/feature_ranking_2D.py

```diff
@@ -0,0 +1,22 @@
+import matplotlib.pyplot as plt
+from sklearn_evaluation.plot import Rank2D
+
+from sklearn.datasets import load_breast_cancer as load_data
+
+X, y = load_data(return_X_y=True)
+
+features = [
```

Comment:
Most of it is duplicated from 1D right?
Maybe it's worth combining the examples?

Reply:
The example scripts are rendered in the docs, like this:
```
Examples
    ---------
    .. plot:: ../../examples/feature_ranking_1D.py
```

So examples are separate for 1D and 2D